### DISCUSSION TITLE
Reset master password if i know admin token

### DISCUSSION THREAD

**Discussion by DavidCroitoru** [dani-garcia/vaultwarden#4911] [Q&A]
I want to change a master pass for an user. I have acces to the admin panel but i dont have an organisation and i dont see the option to create one. 
I want or to change the password or to export the vault(even via terminal) with the acces token.

**Comment by BlackDex** [comment]
That is not possible. Either the user must have been enrolled into there master password reset via an organisation. Or he must have configured an emergency access contact. No other way to achieve access.

**Reply by DavidCroitoru** [reply]
Thanks for replying so quick. I managed to remember the pass:), but how do i create an organisation?

**Comment by BlackDex** [comment]
https://bitwarden.com/help/getting-started-organizations/#setup-your-organization

**Comment by Jaydenha09** [comment]
bitwarden is designed that you dont have to trust the server, being a server operator doesnt mean you should be able to read your user's secret
